src/views/TableView/TableView.js

```diff
@@ -0,0 +1,31 @@
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import { compose } from "recompose";
```

Comment:
This library was discontinued in 2018 and most of the functionality it offers is now in the hooks API. Would you rewrite this to use the hook API if the React version was bumped?

Reply:
Yeah, no problem going forward with Hooks, been using them for the past three months and I've really enjoyed the way they work.